MVVM/Model/PressedKeys.cs

```diff
@@ -212,10 +224,51 @@ private void HandleControlModeChange()
 			StopAll();
 		}
 
+		private void HandleDriveModeChangeToggle()
+		{
+			if (LocalSettings.Singleton.Joystick.RoverDriveController != 3) return;
```

Comment:
Remove "magic number" and use value from `RoverDriveControllerSelector.Controller` enum
https://github.com/wisniax/RoverControlApp/blob/3b3e3cfab920852433be6b94e847b16dd4c70398/Core/RoverControllerPresets/RoverDriveControllerSelector.cs#L8-L14
```suggestion
			if (LocalSettings.Singleton.Joystick.RoverDriveController != (int)RoverDriveControllerSelector.Controller.DirectDrive) return;
```

Reply:
Adressed in [5ed280b](https://github.com/wisniax/RoverControlApp/pull/36/commits/5ed280b9da4555ee40b428e3d684a0ecffc34549)